rero_ils/modules/documents/api.py

```diff
@@ -45,6 +47,22 @@
 document_id_fetcher = partial(id_fetcher, provider=DocumentProvider)
 
 
+class DocumentsIndexer(IlsRecordIndexer):
+    """Indexing documents in Elasticsearch."""
+
+    def index(self, record):
+        """Index an document."""
+        return_value = super(DocumentsIndexer, self).index(record)
+        from ..persons.api import Person
+        for author in record.replace_refs().get('authors', []):
```

Comment:
An idea: Can the index of authors become optional (default=Y) ? I am thinking when there is a mass correction at the document level that does not touch the authorities. This way we can bypass the authority indexing.

Reply:
Mass corrections can use bulk indexing.